apps/backend/src/awsConfig/dynamoDBClient.ts

```diff
@@ -0,0 +1,9 @@
+import * as AWS from 'aws-sdk';
+import { DocumentClient } from 'aws-sdk/clients/dynamodb';
+import 'dotenv/config';
```

Comment:
we don't another lib to support environment
check it: https://nx.dev/recipes/tips-n-tricks/define-environment-variables